website_and_docs/content/documentation/webdriver/browsers/firefox.en.md

```diff
@@ -120,7 +120,15 @@ driver = RemoteWebDriver(options)
 
 ## Add-ons
 
-Unlike Chrome, Firefox extensions are not added as part of capabilities, they are created after starting the driver.
+Unlike Chrome, Firefox extensions are not added as part of capabilities as mentioned in
+[this issue](https://github.com/SeleniumHQ/selenium/issues/10278),
```

Comment:
I think we should refer to the open mozilla issue instead of the closed Selenium issue https://github.com/mozilla/geckodriver/issues/1476

Reply:
Fixed in 7a7989dabce6c25cdc9c3cf969da1521f3b57924